methods/admin/admin.conversations.setConversationPrefs.json

```diff
@@ -12,7 +12,8 @@
     "prefs": {
       "required": true,
       "desc": "The prefs for this channel in a stringified JSON format.",
-      "type": "string"
+      "type": "string",
+      "json_encoded_string": true
```

Comment:
The docs for `admin.conversations.setConversationPrefs` say "The prefs for this channel in a stringified JSON format.". Let's be future proof/more deliberate and instead add a `"format": "json"` instead?